### PR TITLE
Set12| Add Node Documentation for...(Documentation Added: 22 nodes)

### PR DESCRIPTION
**Pull request by johnpierson**
### Purpose
Final PR for this quarter.

### Declarations
DSRevitNodesUI.FilledRegionTypes.dyn
Revit.Elements.FilledRegion.ByCurves.dyn
Revit.Elements.FilledRegionType.Color.dyn
Revit.Elements.FilledRegionType.FillPatternId.dyn
Revit.Elements.Group.GetMembers.dyn
Revit.Elements.Group.GroupType.dyn
Revit.Elements.Group.Location.dyn
Revit.Elements.Group.UngroupElements.dyn
Revit.Elements.Tag.GetLeaderEndCondition.dyn
Revit.Elements.Tag.HeadLocation.dyn
Revit.Elements.Tag.LeaderEnd.dyn
Revit.Elements.Tag.TaggedElement.dyn
Revit.Elements.Tag.TagText.dyn
Revit.Elements.TextNote.ByLocation.dyn
Revit.Elements.TextNote.Height.dyn
Revit.Elements.TextNote.HorizontalAlignment.dyn
Revit.Elements.TextNote.SetText.dyn
Revit.Elements.TextNote.Text.dyn
Revit.Elements.TextNote.Typename.dyn
Revit.Elements.TextNote.VerticalAlignment.dyn
Revit.Elements.TextNoteType.ByName.dyn
Revit.Elements.TextNoteType.Default.dyn
  
Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [x] Snapshot of UI changes, if any.

### Reviewers
@achintyabhat 

### FYIs
@Amoursol

### CODE REVIEW

File: doc/distrib/NodeHelpFiles/_review/DSRevitNodesUI.FilledRegionTypes.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## In Depth
+`Select Filled Region Type` allows for the selection of a filled region type from all types available in the active Revit document.
+
+In the example below, a new drafting view is created in the active Revit document. A filled region is constructed in the view of the "Diagonal Down" type.
```
Comment (achintyabhat) [inline doc/distrib/NodeHelpFiles/_review/DSRevitNodesUI.FilledRegionTypes.md:4]: Rephrase last sentence for clarity : "A filled region is created in the view using "Diagonal Down" type."

File: doc/distrib/NodeHelpFiles/_review/Revit.Elements.TextNote.Height.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+## In Depth
+`TextNote.Height` returns the given text note's height in project units.
+
+In the example below, all working notes are collected from the current Revit document. The height values are then returned.
```
Comment (achintyabhat) [inline doc/distrib/NodeHelpFiles/_review/Revit.Elements.TextNote.Height.md:4]: What is the unit of these values?

File: doc/distrib/NodeHelpFiles/_review/Revit.Elements.TextNote.SetText.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+## In Depth
+`TextNote.SetText` sets the given text note to the given value.
```
Comment (achintyabhat) [inline doc/distrib/NodeHelpFiles/_review/Revit.Elements.TextNote.SetText.md:2]: Rephrase for clarity: TextNote.SetText sets the given text value to the text note.

**Review by achintyabhat — CHANGES_REQUESTED** [review]
Minor changes required for 4 node help documentation.

File: doc/distrib/NodeHelpFiles/_review/Revit.Elements.TextNote.Height.md (hunk 2 above)
Comment (johnpierson) [reply doc/distrib/NodeHelpFiles/_review/Revit.Elements.TextNote.Height.md]: Added more info

Comment (johnpierson) [reply doc/distrib/NodeHelpFiles/_review/Revit.Elements.TextNote.Height.md]: `TextNote.Height` returns the given text note's height in project units.

In the example below, all working notes are collected from the current Revit document. The height values are then returned in project units _(feet and inches)_.

**Review by achintyabhat — APPROVED** [review]

**Pull request merged**